docs/topics/wildfly-swarm/docs/reference/fractions/management.adoc

```diff
@@ -255,6 +267,9 @@ Whether targets having this application type constraint are considered applicati
 swarm.management.authorization-access.standard-role-names:: 
 The official names of the standard roles supported by the current management access control provider.
 
+swarm.management.authorization-access.use-identity-roles:: 
+Should the raw roles obtained from the underlying security identity be used directly?
```

Comment:
remove the question mark

Reply:
Note to self: not easily fixable in our sources.